examples/simple-game-server/main.go

```diff
@@ -322,26 +323,23 @@ func handleResponse(txt string, s *sdk.SDK, cancel context.CancelFunc) (response
 			responseError = fmt.Errorf("Invalid INCREMENT_COUNTER, should have 2 arguments")
 			return
 		}
-		response, responseError = incrementCounter(s, parts[1], parts[2])
-		addACK = false
```

Comment:
I haven't made any modifications to the value of this `addACK`.

Here's the code snippet:

```
if err != nil {
		return err.Error(), false, err
	}
```

Reply:
That's in the case where there is an error. In the case where there is no error it returns the default `addACK = true`.

Reply:
Yes I understand this part, when there is no error, the default value will be `addAck = true`.
Sorry, I want to make sure I get it right. Could you please share your suggestion differently?

Reply:
I believe @igooch is saying that when you look at the e2e tests:

```
    fleet_test.go:1745: 
        	Error Trace:	/go/src/agones.dev/agones/test/e2e/fleet_test.go:1745
        	Error:      	Not equal: 
        	            	expected: "SUCCESS: true\n"
        	            	actual  : "ACK: SUCCESS\n"
```

The reason that the actual response is `ACK: SUCCESS`, is because you removed the line `addACK = false` -- whereas it should remain in to remove the "ACK: " from the response we're seeing in e2e tests (and I expect locally as well).

Reply:
That's a good pickup @igooch - I didn't see the "ACK: SUCCESS" output in https://github.com/googleforgames/agones/pull/3738#discussion_r1559991041 - whereas the output should just be "SUCCESS".

Reply:
Thank you for explaining. I've made changes to the code in the main.go and gameserver_test.go files. Please review the testing for the success use case. Once you both confirm the changes, will push a new image to production.

```
$ nc -u localhost 7654
GET_COUNTER_COUNT rooms
COUNTER: 1
INCREMENT_COUNTER rooms 1
SUCCESS
INCREMENT_COUNTER rooms 1
SUCCESS
DECREMENT_COUNTER rooms 1
SUCCESS
SET_COUNTER_COUNT rooms 5
SUCCESS
SET_COUNTER_CAPACITY rooms 6
SUCCESS
SET_LIST_CAPACITY players 1000
SUCCESS
APPEND_LIST_VALUE players test3
SUCCESS
DELETE_LIST_VALUE players test2
SUCCESS
```

Reply:
That looks good!

Reply:
@igooch I look forward to hearing your thoughts at your convenience!🙂